README.md

```diff
@@ -20,9 +20,8 @@ By the end of the boot camp, all the boot campers should get a clear overview of
 
 - General Recommendation: Learn Rust from Day 0. 
 - zustlings
-- Reading?
-
-// BLOCKCHAIN OVERVIEW
+- Reading
```

Comment:
Hehe, remove the Reading and blockchain overview from here. zustlings too.

also, add good rust resources you know of. rust book, rustlings, lets get rusty....